client/public/styles.css

```diff
@@ -55,3 +59,12 @@ main {
   padding: 16px;
   margin: auto;
 }
+
```

Comment:
Try to write the css in separate files and not alter the main css file.

Reply:
Great, I added a new pull request so it would be to the dev branch. I also forgot about the CSS carousel update. Feel free to delete this pull request.